__tests__/app.test.js

```diff
@@ -0,0 +1,34 @@
+const seed = require('../db/seeds/seed.js');
+const {articleData, commentData, topicData, userData}   = require('../db/data/test-data');
+const db = require('../db/connection');
+const request = require('supertest');
+const app = require('../db/app.js');
+
+beforeEach(() => {
+    return seed({articleData, commentData, topicData, userData});
+  });
+  
+  afterAll(() => db.end());
+
+  describe('NC-News',() => {
+    describe('Get api/topics', () => {
+      test('Responds with 200 and array of objects', () => {
+return request(app).get("/api/topics").expect(200).then(({body: {topics}}) => {
+topics.forEach((topic) => {
+  expect(topic).toHaveProperty("description")
+  expect(topic).toHaveProperty("slug")
+        })
+    })
+})
+})
+})
+
+  describe('NC-News',() => {
+  describe('Bad path api/topics', () => {
```

Comment:
this bad path check looks like it could be applied anywhere - any endpoint that is incorrectly typed is an invalid path so this test could be bigger than this - maybe a 'Routes that don't exist' describe block

Reply:
I would also stay away from having a 'bad paths' describe block for an endpoint and a 'good paths' describe

Keep all test blocks (good and bad) in the same describe block if they're for the same endpoint.